java/src/Ice/src/main/java/com/zeroc/IceInternal/ProxyOutgoingAsyncBaseI.java

```diff
@@ -303,37 +305,184 @@ protected boolean finished(boolean ok, boolean invoke) {
     return super.finished(ok, invoke);
   }
 
-  protected int handleException(com.zeroc.Ice.Exception exc) {
-    Holder<Integer> interval = new Holder<>();
-    _cnt = _proxy._handleException(exc, _handler, _mode, _sent, interval, _cnt);
-    return interval.value;
+  protected int handleException(com.zeroc.Ice.Exception ex) {
```

Comment:
The name handleException will be problematic in C++ and C#, since the base interface has lots of handleXxx, including handleException. So we should find a better name to avoid confusion.